includes/wc-stock-functions.php

```diff
@@ -83,7 +83,8 @@ function wc_maybe_reduce_stock_levels( $order_id ) {
  * @param int $order_id
  */
 function wc_reduce_stock_levels( $order_id ) {
-	$order = wc_get_order( $order_id );
+	$order      = wc_get_order( $order_id );
+	$data_store = WC_Data_Store::load( 'order' );
```

Comment:
How about using `->get_data_store()` here in case it's a special order type or something?